backend/benefit/calculator/tests/test_calculator_api.py

```diff
@@ -381,3 +381,14 @@ def test_unassign_handler_invalid_status(
         data,
     )
     assert response.status_code == 400
+
+
+def test_application_retrieve_calculation_rows_hl422(
```

Comment:
Would e.g. `test_application_calculation_rows_id_exists` be more descriptive?

Reply:
hmm yes it would, changed